OpenTESArena/src/Entities/Camera3D.cpp

```diff
@@ -6,8 +6,8 @@
 #include "../Math/Constants.h"
 #include "../Math/Quaternion.h"
 
-Camera3D::Camera3D(const Double3 &position, const Double3 &direction)
-	: position(position), forward(direction)
+Camera3D::Camera3D(const Double3 &direction, const Double3 &position)
```

Comment:
If not, it's not a big deal. If I understand correctly the warning is just to keep you from mistakenly thinking the variables are being initialized in the order of the initialization list, when they are in fact initialized in the order of their declaration.